### ISSUE TITLE
gh-pages build failed

### ISSUE THREAD

**Issue by byxorna** [tumblr/collins#518]
Looks like gh-pages build failed due to 86135ee70caf2aaeda8491ec72f53e7857b8c30f https://github.com/tumblr/collins/pull/508

```
The page build failed with the following error:

There was a YAML syntax error on line 1751 column 66 in `_data/api.yml`: `mapping values are not allowed in this context`. For more information, see https://help.github.com/articles/page-build-failed-invalid-yaml-in-data-file/.

For information on troubleshooting Jekyll see:

  https://help.github.com/articles/troubleshooting-jekyll-builds

If you have any questions you can contact us by replying to this email.
```

**Comment by ssgelm** [comment]
Oh, now that I added a colon in line 1751 that string needs to be quoted.  Would you like me to do that or will you?

**Comment by byxorna** [comment]
@ssgelm ill take care of it :)

**Comment by byxorna** [comment]
All fixed! :) #520 filed to hook up travis builds for gh-pages
